packages/api/amplify_api/example/integration_test/graphql/iam_test.dart

```diff
@@ -0,0 +1,334 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+import 'dart:convert';
+
+import 'package:amplify_api/amplify_api.dart';
+import 'package:amplify_api_example/models/ModelProvider.dart';
+import 'package:amplify_flutter/amplify_flutter.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+import '../util.dart';
+
+void main({bool useExistingTestUser = false}) {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  group('GraphQL IAM', () {
+    setUpAll(() async {
+      await configureAmplify();
+
+      if (!useExistingTestUser) {
+        await signUpTestUser();
+      }
+      await signInTestUser();
+    });
+
+    tearDownAll(() async {
+      await deleteTestModels();
+      if (!useExistingTestUser) {
+        await deleteTestUser();
+      }
+    });
+
+    group('queries (authorized user)', () {
+      setUpAll(() async {
+        await signInTestUser();
```

Comment:
This will be called twice since `setUpAll`s stack when nesting groups - true for all setUp/tearDown methods

Reply:
I might be missing something but I debugged this and only saw it called once so may sync offline.

Reply:
gotcha. My mistake, I was debugging the wrong test file when I claimed only called once. This was leftover from previous organization of file and removing now, ty.